src/Illuminate/Filesystem/FilesystemAdapter.php

```diff
@@ -240,7 +240,7 @@ public function getVisibility($path)
      *
      * @param  string  $path
      * @param  string  $visibility
-     * @return void
+     * @return bool
```

Comment:
This is wrong. The contract states that this should return void. https://github.com/laravel/framework/blob/bd352a0d2ca93775fce8ef02365b03fc4fb8cbb0/src/Illuminate/Contracts/Filesystem/Filesystem.php#L64